compiler/modules/types/list.bal

```diff
@@ -132,6 +132,10 @@ function listSubtypeIsEmpty(Context cx, SubtypeData t) returns boolean {
     return memoSubtypeIsEmpty(cx, cx.listMemo, listBddIsEmpty, <Bdd>t);
 }
 
+function listSubtypeIsCyclic(Context cx, SubtypeData t) returns boolean {
```

Comment:
Better to be listBddIsCyclic and take a Bdd, similarly for mapping.

Reply:
fixed